bot/pagination.py

```diff
@@ -218,10 +222,10 @@ async def paginate(
         def event_check(reaction_: discord.Reaction, user_: discord.Member) -> bool:
             """Make sure that this reaction is what we want to operate on."""
             no_restrictions = (
-                # Pagination is not restricted
-                not restrict_to_user
```

Comment:
This removes support for unrestricted pagination. Now, all pagination must either be restricted to the given user or staff. Was this not a feature we relied upon?

Reply:
As far as I can see, defaulting to `ctx.author` won't break anything, the features using the paginator should still work correctly.